lib/src/components/NxModal/NxModal.tsx

```diff
@@ -42,7 +42,7 @@ const NxModal: FunctionComponent<Props> = ({className, onClose, variant, ...attr
     currentModalCloseHandlers.push(onClose);
 
     return removeCloseHandlerListener;
-  }, []);
+  }, [onClose]);
```

Comment:
I remember getting a warning because I wasn't using all received props here, is not present anymore? (memory fails if it was in the IDE or in the console)

Reply:
> I remember getting a warning because I wasn't using all received props here, is not present anymore? (memory fails if it was in the IDE or in the console)

I haven't gotten that warning nor do I see how it would make sense on this line.